app/controllers/partnering_organizations_controller.rb

```diff
@@ -17,6 +17,15 @@ def index
     end
   end
 
+ #GET /partnering_organization/subscriptions
+ def subscriptions
```

Comment:
So, if you wanted to make this work, you'd have to make a `member` route in `routes.rb` (so that it passes the PO id) and then you'd make it `@current_partnering_organizations.subscriptions` and add an end to the `respond_to` but I think you should just pass the subscribers through the view (I'll explain later in the review)